pkg/network/image-pull-policy.go

```diff
@@ -40,6 +44,11 @@ func changeSafeImagePullPolicy(prev, next *opv1alpha1.NetworkAddonsConfigSpec) [
 	return []error{}
 }
 
+// Currently not implemented, since there are no obsolete objects under this module
+func CleanUpImagePullPolicy(ctx context.Context, client k8sclient.Client, objs []*unstructured.Unstructured) []error {
```

Comment:
No need to add this to every module. Add it only to those implementing it and then collect them in network.go

Reply:
what do you mean by this?

Reply:
Look at the other functions (such as Validate), only modules implementing it have it. Others are simply not executed from network.go